lib/plenario_web/controllers/api/method_not_allowed_controller.ex

```diff
@@ -11,9 +11,13 @@ defmodule PlenarioWeb.Api.MethodNotAllowedController do
       |> Enum.drop(1)))
 
     if path_info in routes do
-      send_resp(conn, 405, "Method not allowed")
+      conn
+      |> put_req_header("accept", "application/vnd.api+json")
```

Comment:
Um... we're not supporting JSON API. We're aping some of its structure, but definitely not using it. We only accept `application/json`.

Reply:
This is so the browser can render the error message, users that are typing in links or using a browser based client.

Reply:
Browsers can handle JSON just fine. Also, the API is absolutely not intended to be accessed by a browser anyway -- if some schmuck is doing it that way, they deserve whatever they get back.